### PR TITLE
change(style/colors): update colors and add descriptions

### PR DESCRIPTION
**Pull request by kirill-kruchkov**
## Purpose
Update with new colours from design-team.

### Visualization
screenshot 2018-11-12 at 15 38 12
screenshot 2018-11-12 at 15 38 17
screenshot 2018-11-12 at 15 44 29

### Checklist
- [ ] Update changelog
- [ ] Validate with the design team

### CODE REVIEW

File: src/components/DropdownItem/DropdownItem.js [hunk 1]
```diff
@@ -4,11 +4,11 @@ import styled from 'styled-components';
 
 const Wrapper = styled.div`
   padding: 0.5rem 1rem;
-  color: ${props => props.theme.darkest};
+  color: ${props => props.theme.grey01};
```
Comment (frontalicious-martijn) [inline src/components/DropdownItem/DropdownItem.js]: Can you go a global find/replace of props and make it 'p' for styled components?

File: src/components/Table/__snapshots__/Table_snapshot.spec.js.snap [hunk 2]
```diff
@@ -93,7 +93,7 @@ ShallowWrapper {
     "props": Object {
       "alternateColors": true,
       "alternateRowColor": "#f7f8fa",
-      "borderColor": "#9b9b9b",
+      "borderColor": "#999",
```
Comment (frontalicious-martijn) [inline src/components/Table/__snapshots__/Table_snapshot.spec.js.snap]: Shall we use `enzyme-to-json` in this project too?

File: src/style/colors/colors.md [hunk 3]
```diff
@@ -1,7 +1,17 @@
...
 const theme = require('../../theme/colors').default;
-<div style={{ display: 'flex', flexDirection: 'row', flexWrap: 'wrap' }}>
-  {Object.keys(theme).map(color => <Colors name={color} value={theme[color]} key={color} />)}
+const usage = require('../../theme/colors').usage;
+<div
+  style={{
+    display: 'flex',
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    justtifyContent: 'space-between',
```
Comment (frontalicious-martijn) [inline src/style/colors/colors.md]: typo

File: src/components/DropdownItem/DropdownItem.js (hunk 1 above)
Comment (zoheiry) [reply src/components/DropdownItem/DropdownItem.js]: `\${props => (.*?)props(.*?)}`
Replace with
`\${p => $1p$2}`

File: src/components/Table/__snapshots__/Table_snapshot.spec.js.snap (hunk 2 above)
Comment (kirill-kruchkov) [reply src/components/Table/__snapshots__/Table_snapshot.spec.js.snap]: We should. Also Prettier

File: src/components/DropdownItem/DropdownItem.js (hunk 1 above)
Comment (kirill-kruchkov) [reply src/components/DropdownItem/DropdownItem.js]: Will do!

**Comment by kirill-kruchkov** [comment]
@frontalicious-martijn thank you for the review!

File: src/components/DropdownItem/DropdownItem.js (hunk 1 above)
Comment (kirill-kruchkov) [reply src/components/DropdownItem/DropdownItem.js]: Actually, I'll better do it in a separate PR because it's too lot of changes to add to this one

**Pull request merged**